src/settings.py

```diff
@@ -57,8 +57,10 @@ class GlobalSettings:
     random_options_max_tries = 100
 
     # SeSamE parameters
+    propagate_failures = True
+    max_samples_per_step = 10
+    max_num_steps_option_rollout = 1000
```

Comment:
can you make this environment-specific? (see below in this file for examples)

Reply:
should I do this even after deleting the first two params?

Reply:
yeah I was only referring to `max_num_steps_option_rollout`, which needs to be larger for behavior than for the other envs